File: backend/src/main/java/com/example/shoestore/dto/response/CheckoutCartItem.java
package com.example.shoestore.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CheckoutCartItem {
    private Integer cartItemId;
    private Integer variantId;
    private String imageUrl;
    private String brand;
    private String productName;
    private String color;
    private String size;
    private Integer quantity;
    private Double price;
}

File: android/app/src/main/java/com/example/shoeshop/dto/respone/CheckoutCartItem.kt
package com.example.shoeshop.dto.respone

data class CheckoutCartItem(
    val cartItemId: Int,
    val variantId: Int,
    val imageUrl: String,
    val brand: String,
    val productName: String,
    val color: String,
    val size: String,
    val quantity: Int,
    val price: Double
)
